proto/binlog/cistern.proto

```diff
@@ -60,3 +64,9 @@ message GetLatestCommitTSResp {
     // isSynced specifies whether the all binlogs are consumed from pump
     bool isSynced = 2;
 }
+
+message NotifyReq {
```

Comment:
the protobuf don't support service message without argument, and maybe we would extend our message in the future.